### PR TITLE
Vatanabe hw 2

### PR DESCRIPTION
**Pull request by DennisVatanabe**
@IlyaSenkevich , i'm waiting for verification

### CODE REVIEW

File: hw_2.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+let a = 1;
+let b = 2;
+let c = "белых медведей";
+
+const resultOneForTaskFour = String(a) + b + ' ' + c;
+const resultTwoForTaskFour = a + (b + ' ' + c);
+
+
+console.log('первый вариант: ', resultOneForTaskFour,
```
Comment (IlyaSenkevich19) [inline hw_2.js:78]: Make it using ``, instead of /n

File: hw_2.js [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+console.log(`Variable: %firstV% have type: %${typeof firstV}%
+    \n Variable: %secondV% have type: %${typeof secondV}%
+    \n Variable: %thirdV% have type: %${typeof thirdV}%
+    \n p.s. не понимаю - нах символы процентов в тз? но тз есть тз`);
+
+// #### Task 7 💻
+
+// Запросить у пользователя имя и возраст и вывести их в консоль.
+
+document.addEventListener("DOMContentLoaded", function() {
```
Comment (IlyaSenkevich19) [inline hw_2.js:123]: It is cool, but it should be using 'prompt'

File: hw_2.js [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+// Создать 3 переменные разных типов и вывести в консоль для каждой из них строку следующего вида:    
+
+//     `Variable: %variable name% have type: %type variable%`
+
+const firstV = true;
+const secondV = 'bmx';
+const thirdV = 2020;
+
+console.log(`Variable: %firstV% have type: %${typeof firstV}%
+    \n Variable: %secondV% have type: %${typeof secondV}%
```
Comment (IlyaSenkevich19) [inline hw_2.js:114]: you can also make it without \n, `` brackets work without it (\n)

Comment (DennisVatanabe) [reply hw_2.js]: Ok

**Pull request merged**